### DISCUSSION TITLE
Inactive status

### DISCUSSION THREAD

**Discussion by wantdead** [DependencyTrack/dependency-track#3600] [Q&A] [answered]
After what time (or by what triggers) does the dependency track put the project on inactive status?

**Comment by nscuro** [comment, marked as answer]
It never does that by itself. Whether a project is active or not is an active decision by users / operators of DT. You can set the activity status manually via UI, or in an automated fashion via REST API.

**Reply by wantdead** [reply]
@nscuro thank you, it helped me understand how to organize project management
